neutron/convert.cpp

```diff
@@ -39,7 +39,7 @@ T convert_generic(T &&d, const Dim from, const Dim to,
       // Cannot realign if any item has aligned (histogrammed) data
       if (!coord.dims().contains(arg.dims()))
         d.coords().set(from, broadcast(coord, arg.dims()));
-      transform_in_place(coord, arg, op_);
+      transform_in_place(d.coords()[from], arg, op_);
```

Comment:
Several of the existing unit tests failed. I am not sure possible to test for this directly because it was probably accessing deallocated memory, etc.